tests/func/parsing/test_interpolated_entry.py

```diff
@@ -259,3 +259,38 @@ def test_vars_load_partial(tmp_dir, dvc, local, vars_):
         d["vars"] = vars_
     resolver = DataResolver(dvc, tmp_dir.fs_path, d)
     resolver.resolve()
+
+
+def test_cmd_dict(tmp_dir, dvc):
+    data = {
```

Comment:
> I ignored it for now (nothing gets added to the interpolated string) .
> Maybe `--no-{x}` makes sense (i.e. is also the [default used in click](https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags)) , the worst thing that can happen is that we will add a flag that it's ignored?

No strong preference here.

> Not sure if we should raise an explicit exception if we detect a nested list or just render as if it was a string, which will probably raise an exception on the user code when parsing the args.

If easy enough, I would prefer the former, but doesn't need to block the PR.

Reply:
> No strong preference here.

Added the `--no-` prefix for negative booleans instead of ignoring them.

> If easy enough, I would prefer the former, but doesn't need to block the PR.

Added explicit exception